tools/collect-dependencies.js

```diff
@@ -0,0 +1,34 @@
+const { execSync } = require('child_process');
+const { writeFileSync } = require('fs');
+const { argv, exit } = require('process');
+
+const option = argv[3];
+const filename = argv[4];
+
+if (option !== '-o' || !filename) {
+  console.log(`command usage: ${__filename} -o filename`);
+  exit(-1);
```

Comment:
I think '9' is more appropriate here. https://nodejs.org/api/process.html#process_exit_codes
```suggestion
  exit(9);